### PR TITLE
Update Runtime links for removal of v1

### PR DESCRIPTION
**Pull request by Eric-Arellano**

### CODE REVIEW

File: docs/migration-guides/qiskit-algorithms-module.mdx [hunk 1]
```diff
@@ -5,6 +5,14 @@ description: How to update your code to use the new interface for `qiskit.algori
 
 # Algorithms migration guide
 
+<Admonition type="caution">
+  **Deprecation Notice**
+
+  This guide precedes the introduction of the V2 primitives interface. Following the introduction of the V2 primitives, some providers have deprecated V1 primitive
+  implementations in favor of the V2 alternatives. If you are interested in following this guide, we recommend combining it with the
+  [migrate to V2 primitives](./v2-primitives) guide to bring your code to the most updated state.
```
Comment (abbycross) [inline docs/migration-guides/qiskit-algorithms-module.mdx:13]:
```suggestion
  [Migrate to V2 primitives](./v2-primitives) guide to bring your code to the most updated state.
```

File: docs/migration-guides/qiskit-algorithms-module.mdx [hunk 2]
```diff
@@ -86,7 +94,7 @@ Once you know which primitive you want to use, choose the primitive implementati
 
 * For running on quantum hardware choose from these options:
 
-   - Access services with native primitive implementations, such as the Qiskit Runtime service by using [`qiskit_ibm_runtime.Sampler`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Sampler) and [`qiskit_ibm_runtime.Estimator`.](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Estimator)
+   - Access services with native primitive implementations, such as the Qiskit Runtime service by using [`qiskit_ibm_runtime.Sampler`](/api/qiskit-ibm-runtime/0.26/qiskit_ibm_runtime.Sampler) and [`qiskit_ibm_runtime.Estimator`.](/api/qiskit-ibm-runtime/0.26/qiskit_ibm_runtime.Estimator)
```
Comment (abbycross) [inline docs/migration-guides/qiskit-algorithms-module.mdx:97]:
```suggestion
   - Access services with native primitive implementations, such as the Qiskit Runtime service, by using [`qiskit_ibm_runtime.Sampler`](/api/qiskit-ibm-runtime/0.26/qiskit_ibm_runtime.Sampler) and [`qiskit_ibm_runtime.Estimator`.](/api/qiskit-ibm-runtime/0.26/qiskit_ibm_runtime.Estimator)
```

File: docs/migration-guides/qiskit-opflow-module.mdx [hunk 3]
```diff
@@ -7,6 +7,14 @@ description: How to update your code to stop using the deprecated `qiskit.opflow
 
 # Opflow migration guide
 
+<Admonition type="caution">
+  **Deprecation Notice**
+
+  This guide precedes the introduction of the V2 primitives interface. Following the introduction of the V2 primitives, some providers have deprecated V1 primitive
+  implementations in favor of the V2 alternatives. If you are interested in following this guide, we recommend combining it with the
+  [migrate to V2 primitives](./v2-primitives) guide to bring your code to the most updated state.
```
Comment (abbycross) [inline docs/migration-guides/qiskit-opflow-module.mdx:15]:
```suggestion
  [Migrate to V2 primitives](./v2-primitives) guide to bring your code to the most updated state.
```

File: docs/migration-guides/qiskit-opflow-module.mdx [hunk 4]
```diff
@@ -22,7 +30,7 @@ the [`qiskit.opflow`](/api/qiskit/0.46/opflow) module to the [`qiskit.primitives
...
-  can be replaced with instances of any primitive implementation. For example Aer primitives ([`qiskit_aer.primitives.Sampler`](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Sampler.html)/[`qiskit_aer.primitives.Estimator`](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Estimator.html)) or the Qiskit Runtime primitives ([`qiskit_ibm_runtime.Sampler`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Sampler)/[`qiskit_ibm_runtime.Estimator`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Estimator)).
+  can be replaced with instances of any primitive implementation. For example Aer primitives ([`qiskit_aer.primitives.Sampler`](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Sampler.html)/[`qiskit_aer.primitives.Estimator`](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Estimator.html)) or the Qiskit Runtime primitives ([`qiskit_ibm_runtime.Sampler`](/api/qiskit-ibm-runtime/0.26/qiskit_ibm_runtime.Sampler)/[`qiskit_ibm_runtime.Estimator`](/api/qiskit-ibm-runtime/0.26/qiskit_ibm_runtime.Estimator)).
```
Comment (abbycross) [inline docs/migration-guides/qiskit-opflow-module.mdx:33]:
```suggestion
  can be replaced with instances of any primitive implementation. For example, Aer primitives ([`qiskit_aer.primitives.Sampler`](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Sampler.html)/[`qiskit_aer.primitives.Estimator`](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Estimator.html)) or the Qiskit Runtime primitives ([`qiskit_ibm_runtime.Sampler`](/api/qiskit-ibm-runtime/0.26/qiskit_ibm_runtime.Sampler)/[`qiskit_ibm_runtime.Estimator`](/api/qiskit-ibm-runtime/0.26/qiskit_ibm_runtime.Estimator)).
```

File: docs/migration-guides/qiskit-quantum-instance.mdx [hunk 5]
```diff
@@ -11,7 +11,7 @@ description: Stop using the deprecated Qiskit `QuantumInstance` class
   This guide precedes the introduction of the V2 primitives interface. It was created to guide users from `QuantumInstance` to
   the V1 primitives interface. Following the introduction of the V2 primitives, some providers have deprecated V1 primitive
   implementations in favor of the V2 alternatives. If you are interested in following this guide, we recommend combining it with the
-  [migrate to V2 primitives] guide to bring your code to the most updated state.
+  [migrate to V2 primitives](./v2-primitives) guide to bring your code to the most updated state.
```
Comment (abbycross) [inline docs/migration-guides/qiskit-quantum-instance.mdx:14]:
```suggestion
  [Migrate to V2 primitives](./v2-primitives) guide to bring your code to the most updated state.
```

File: docs/migration-guides/qiskit-algorithms-module.mdx (hunk 1 above)
Comment (Eric-Arellano) [reply docs/migration-guides/qiskit-algorithms-module.mdx:13]: Why would this be capitalized? It's part of a sentence

**Review by abbycross — APPROVED** [review]
👍

File: docs/migration-guides/qiskit-algorithms-module.mdx (hunk 1 above)
Comment (abbycross) [reply docs/migration-guides/qiskit-algorithms-module.mdx:13]: The guide name is capitalized so we maintain that capitalization. See for example:
Screenshot 2024-08-16 at 1 38 39 PM

File: docs/migration-guides/qiskit-algorithms-module.mdx [hunk 6]
```diff
@@ -548,7 +556,7 @@ print(result.eigenvalues)
 **[Updated] Using primitives:**
 
 ```python
-from qiskit_ibm_runtime import Sampler, Estimator, QiskitRuntimeService, Session
+from qiskit_ibm_runtime import SamplerV1 as Sampler, EstimatorV1 as Estimator, QiskitRuntimeService, Session
```
Comment (Eric-Arellano) [inline docs/migration-guides/qiskit-algorithms-module.mdx:559]:
```suggestion
from qiskit_ibm_runtime import Sampler, Estimator, QiskitRuntimeService, Session
```

**Pull request merged**